target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -158,6 +158,9 @@ vr500)
 dir-860l-b1)
 	ucidef_set_led_netdev "wan" "wan" "$board:green:net" "eth0.2"
 	;;
+dwr-921-c1)
+	ucidef_set_led_netdev "lan" "lan" "$board:green:lan" "eth0.1"
```

Comment:
Please use ucidef_set_led_switch for the lan led. The argument to ucidef_set_led_switch is a binary mask, where the most right bit is switch port 0.